src/features/cartographie/layouts/cartographie/cartographie.layout.providers.ts

```diff
@@ -11,7 +12,7 @@ export const cartographieLayoutProviders = [
     useClass: LieuxMediationNumeriquePresenter
   },
   {
-    deps: [LieuxMediationNumeriqueRepository],
+    deps: [LieuxMediationNumeriqueRepository, HttpClient],
```

Comment:
dépendance```suggestion
    deps: [LieuxMediationNumeriqueRepository, AccesLibreRepository],
```
Pas d'utilisation du `HttpClient` end dehors d'un repository
Reproduire le même schéma que celui en place pour `LieuxMediationNumeriqueRepository` en créant `AccesLibreRepository`, l'intérêt de faire cela est d'isoler au maximum les dépendances du reste du code.

Le seul intérêt de la présentation c'est uniquement de préparer les données pour l'affichage, en partant du principe que ces données sont déjà récupérées.
La présentation ne sait pas comment les données sont récupérées (json, API, web socket, local storage, etc.).

C'est le rôle du repository de fournir un moyen de récupérer les données : on se sert du principe d'injection de dépendance pour donner une implémentation concrète de la récupération des données, dans le cas présent en utilisant le client Http.